Can't build latest version

#### Category
- [ ] Question
- [X ] Bug
- [ ] Enhancement

#### Expected or Desired Behavior
I want to be able to build the project.

#### Observed Behavior
I downloaded the latest code, unzipped it, ran "npm install" and then tried to build it. I throws 2 errors:
_Error - [tsc] src/webparts/handlebarTemplateDisplay/HandlebarTemplateDisplayWebPart.ts(24,36): error TS1149: File name 'S:/LinksAndHandlebarsTemplate/src/propertyPane/propertyPaneSearch/PropertyFieldSearch.ts' differs from already included file name 'S:/LinksAndHandlebarsTemplate/src/propertyPane/PropertyPaneSearch/PropertyFieldSearch.ts' only in casing.
[11:40:37] Error - 'tsc' sub task errored after 10 s
 exited with code 2_

that appear to be the same error. Is there something I'm missing? 

#### Steps to Reproduce
1. Download code
2. Extract code
3. run "npm install"
4. run "gulp build"

--- a/solutions/LinksAndHandlebarsTemplate/src/webparts/handlebarTemplateDisplay/HandlebarTemplateDisplayWebPart.ts
+++ b/solutions/LinksAndHandlebarsTemplate/src/webparts/handlebarTemplateDisplay/HandlebarTemplateDisplayWebPart.ts
@@ -21,7 +21,7 @@ import { IHandlebarTemplateDisplayWebPartProps } from './IHandlebarTemplateDispl
 import { PropertyFieldCamlQueryFieldMapping, PropertyFieldCamlQueryOrderBy } from "../../propertyPane/propertyFieldCamlQueryFieldMapping/PropertyFieldCamlQueryFieldMapping";
 import QueryStringParser from "../../utilities/urlparser/queryStringParser";
 import pnp, { SearchQueryBuilder, Sort, SearchProperty } from 'sp-pnp-js';
-import { PropertyPaneSearch } from '../../propertyPane/propertyPaneSearch/PropertyFieldSearch';
+import { PropertyPaneSearch } from '../../propertyPane/PropertyPaneSearch/PropertyFieldSearch';
 import { WebPartLogger } from '../../utilities/webpartlogger/usagelogger';
 
 export default class HandlebarTemplateDisplayWebPart extends BaseClientSideWebPart<IHandlebarTemplateDisplayWebPartProps> {

